### ISSUE TITLE
Tried to set charges for item butchery refuse (fresh) that could not have them!

### ISSUE THREAD

**Issue by IdleSol** [CleverRaven/Cataclysm-DDA#73106]
### Describe the bug

```
Ru: DEBUG    : Tried to set charges for item отход от разделки (свежее) that could not have them!
En: DEBUG    : Tried to set charges for item butchery refuse (fresh) that could not have them!

 FUNCTION : item::item(const itype*, time_point, int)
 FILE     : src/item.cpp
 LINE     : 318
 VERSION  : cdda-experimental-2024-04-16-1009 73b6cf7
```

### Attach save file

[Госпорт-trimmed.tar.gz](https://github.com/CleverRaven/Cataclysm-DDA/files/15026578/-trimmed.tar.gz)

### Steps to reproduce

1. Load the save
2. Take 2-3-4 steps on the road
3. Get an error

If the error does not occur, exit without saving and repeat the steps

Object spawn location: underground lab. Directly below the character.

### Expected behavior

N/A

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19043 (21H1)
- Game Version: cdda-experimental-2024-04-16-1009 73b6cf7 [64-bit]
- Graphics Version: Tiles
- Game Language: Русский <color_dark_gray>(90.3%)</color> [ru]
- Mods loaded: [
    Тёмные Дни Впереди [dda],
    Отключить потребности у НПС [no_npc_food],
    Портальные штормы игнорируют НПС [personal_portal_storms],
    Медленный рост фунгалоидов [no_fungal_growth]
]

### Additional context

Probably related to these issues #72984, #73105
see post https://github.com/CleverRaven/Cataclysm-DDA/issues/72984#issuecomment-2064170122

**Comment by PatrikLundell** [comment]
This may have been fixed by #73104, but I cannot be sure. This error message didn't appear after applying that fix, but I haven't seen it myself because the chunks of meat instance cropped up.
It would be fixed if the refuse is produced by the same code as the chunks of meat, but the debugger decided to skip parts of the call stack, so I don't know where the call actually came from, and thus whether the same code was used to spread both chunks of meat and refuse around, but it seems possible, as gibbed monsters do spread both in a similar fashion.

**Comment by IdleSol** [comment]
After updating, 5 attempts out of 5, no errors. I'm closing the topic.
